src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -27,6 +27,14 @@ public class DiscordNotification : BaseNotification<DiscordNotificationSettings>
             Logger = log;
         }
 
+        // constants I needed but could not find
+        public const string IMDB_BASE_URL = "http://www.imdb.com/title/";
+        public const string TVDB_BASE_URL = "https://www.thetvdb.com/?tab=series&id=";
+
+        // if true mentionAlias will post the alias to discord and trigger an @mention if set up as <@id>.  
+        // It will also use the username instead of alias when talking about a user. e.g. "Requested by username on 16 March"
+        public bool mentionAlias { get; set; } = true;
```

Comment:
Will this be settable from the UI?

Reply:
Also since it's a public property it should have a capital M

Reply:
Yes, the idea is to eventually have it be settable via UI once I figure out how to add that. For now now it's true by default.